src/game/stage/rings_scatter.c

```diff
@@ -324,15 +497,26 @@ void RingsScatterSingleplayer_FlippedGravity(void)
         }
 
         {
+#if (GAME == GAME_SA2)
             u16 sprFlags = ring->unk10;
             ring->unk10 &= ~0x3;
             ring->unk10 |= (sprFlags + 1) & 0x3;
+#endif
             ring->unkC--;
         }
     }
 }
+#if (GAME == GAME_SA1)
+END_NONMATCH
+#endif
 
+#if (GAME == GAME_SA1)
+// (98.01%) https://decomp.me/scratch/iOeRj
+NONMATCH("asm/non_matching/game/stage/rings_scatter/RingsScatterSingleplayer_NormalGravity.inc",
```

Comment:
Again don't like the idea of merging non matches together

Reply:
I kinda get it.
It's weird how the same function can be so drastically different between games.

So either find a match for both or remove the SA1 case from here?

Reply:
Yeah, I think until the match is in place it feels wrong to place the non match stuff in here :/

Reply:
Fixed btw. ^^